DOC: Roadmap documentation (#58)
nipreps/dmriprep: README.rst

The workflow is based on `Nipype <https://nipype.readthedocs.io>`__ and encompases a large
set of tools from well-known neuroimaging packages, including
`ANTs <https://stnava.github.io/ANTs/>`__,
`FreeSurfer <https://surfer.nmr.mgh.harvard.edu/>`__,

> and `DIPY <https://dipy.org/>`__.

Reviewer: Should we just not mention any of these here (so long the citations are clearly stated in the boilerplate)?

Author: Yes. I think that would be fine. Implemented in 5431efc

This pipeline was designed to provide the best software implementation for each state of
preprocessing, and will be updated as newer and better neuroimaging software becomes
available.

> dMRIPrep performs basic preprocessing steps (head motion correction,

Reviewer: ```suggestion
dMRIPrep performs basic preprocessing steps (:abbr:`HMC (Head-Motion Correction)`,
```

> susceptibility distortion correction, eddy current correction,  etc.) providing

Reviewer: ```suggestion
:abbr:`SDC (Susceptibility-derived Distortion Correction)`,
:abbr:`ECC (Eddy-Currents Correction)`,  etc.) providing
```

> outputs that can be easily submitted to a variety of tractography algorithms.

Reviewer: ```suggestion
outputs that can be easily submitted to a variety of diffusion models.
```
